ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -849,6 +859,111 @@ function getPoint(
     return point;
 }
 
+/**
+ * @private
+ */
+function scatterProcessData(
+    this: Series,
+    force?: boolean
+): (boolean|undefined) {
+    const series = this,
+        options = series.options,
+        xAxis = series.xAxis,
+        yAxis = series.yAxis;
+    (window as any).TheSeries = series;
```

Comment:
👀

Reply:
PS: A trick I use to stop myself from committing temporary debug code is to write it in a way that doesn't pass lint. That way (and it happens often), I won't be able to commit until I've removed it.